ros_trick/ros_trick_bridge.Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM osrf/space-ros:humble-2024.10.0
+FROM osrf/space-ros:jazzy-2025.04.0
```

Comment:
Any reason not to build this off of the latest `main` image too if we're going to be updating the others?